src/coreComponents/fileIO/Outputs/TimeHistoryOutput.hpp

```diff
@@ -110,7 +110,7 @@ class TimeHistoryOutput : public OutputBase
    * @return Return PyHistoryOutput type.
    */
 #if defined(GEOSX_USE_PYGEOSX)
-  virtual PyTypeObject * getPythonType() const;
+  virtual PyTypeObject * getPythonType() const override;
```

Comment:
We have a strict requirement for this...don't know why it doesn't show up on CI??